airflow/providers/amazon/aws/hooks/eks.py

```diff
@@ -76,7 +76,7 @@ class NodegroupStates(Enum):
     NONEXISTENT = "NONEXISTENT"
 
 
-COMMAND = """
+COMMAND = r"""
             output=$({python_executable} -m airflow.providers.amazon.aws.utils.eks_get_token \
                 --cluster-name {eks_cluster_name} {args} 2>&1)
```

Comment:
This kind of fix already merged and after that reverted

- https://github.com/apache/airflow/pull/38734
- https://github.com/apache/airflow/pull/38864